### ISSUE TITLE
Request: add "edit" link in activity hamburger menu

### ISSUE THREAD

**Issue by mlutfy** [civicrm/org.civicrm.civicase#115]
From the Contact record, activity tab, we can see Overdue and Past Activities. A hamburger menu lets the user delete, move to case or copy to case. However, it does not let the user Edit.

![civicase-activity-edit-link](https://user-images.githubusercontent.com/254741/44814376-86410c00-abaa-11e8-9ab1-4460ceab0cd4.jpg)

In testing we have done, many users complained that they could not easily edit activities. They had to first click on it, then scroll down. When there are a lot of past/overdue activities, this may sound silly, but it becomes cumbersome to scroll to the right place on the screen (there are some fixed-height calculations being done, and I'm thinking of overriding the CSS locally, because I think users get confused by scrolling in the wrong place, especially if they have a tactile screen).

Long story short: could we add an 'edit' link in the hamburger menu?

**Comment by colemanw** [comment]
I would say the fixed height/scrolling issue with the case sidebar is working pretty well on CiviCase screens, but is broken on the contact summary screen "Activities" tab. @jamienovick is this on your radar?
